backend/Match/views.py

```diff
@@ -27,7 +27,7 @@ def get(self, request):
         if token.user_left_side.id != request.user.id:
             return Response({'error': 'You are not the host user in the token'}, status=status.HTTP_403_FORBIDDEN)
 
-        match, player_left, player_right = GameSetupManager.create_match_and_its_players(token)
+        match, player_left, player_right = MatchSetupManager.create_match_and_its_players(token)
 
         if not all([match, player_left, player_right]):
             return Response({'error': 'Something went wrong when creating the match and players'}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
```

Comment:
We need to return something other than 5xx in case something does in fact go wrong during the eval (like run out of space on the school computers or so)

Reply:
changed